### ISSUE TITLE
Incomplete / confusing change log version history

### ISSUE THREAD

**Issue by joshuambg** [Compass/compass#1843]
On Page: http://compass-style.org/CHANGELOG/

I had a problem with a project that was using CSS3 PIE. I recently installed the project on a new machine but CSS3 PIE support had been removed from the latest compass version. I couldn't understand why I didn't have problems before. The latest version was 1.0.1 According to the Change Log page Pie was removed in 0.13.alpha.8. I never installed alpha versions for the project.

This page better demonstrated the discrepancies between versions and release dates.
https://rubygems.org/gems/compass/versions

I was originally using version 0.12.6 released in April 2014. Version 1.0.1 was only released August 2014. All the alpha releases since 2013 is really confusing!!

Version 0.12.6 isn't even listed on the Change Log?? There should be a more extensive and more descriptive version history page like the ruby gems example.

**Comment by scottdavis** [comment]
so 0.12.6 was a legacy bug fix release. 0.12.x was also the last stable release of compass. 0.13 was a development release. with the release of 1.0.0 we have adopted http://semver.org/.

I hope that helps clear things up

**Comment by joshuambg** [comment]
Yes these issues are clear to me now. The specific problem I'm reporting, your official change log page is not intuitive to the logic of version release history. It needs to be redesigned to be easier to understand. Unless you're okay having people google the problem and maybe happen on this github thread...

**Comment by scottdavis** [comment]
0.12.6 should have been on the change log and im sorry it wasn't we have been going through a lot of restructuring internally on this project and it possibly slipped through the cracks.

**Comment by joshuambg** [comment]
It's not just that 0.12.6 was missing. Scrolling down the page it doesn't make sense seeing the 0.13 alpha versions released in 2013 then 0.12 versions released _after_ those. The page needs to be looked at and redesigned.

**Comment by lolmaus** [comment]
> Scrolling down the page it doesn't make sense seeing the 0.13 alpha versions released in 2013 then 0.12 versions released _after_ those.

Wut? Why?

**Comment by scottdavis** [comment]
I'm somewhat confused since the rubygems page you mention is in fact in not the same order you are describing.

```
0.13.alpha.0 May 6, 2012 (1.54 MB)
0.12.7 July 17, 2014 (1.11 MB)
```

Our change log is just a mark down file located here: https://github.com/Compass/compass/blob/stable/compass-style.org/content/CHANGELOG.markdown

Are you proposing that we relocate the 0.12.x releases of 2014 to be mixed in with the 0.13 alpha's ordered by release date?

**Comment by joshuambg** [comment]
Maybe the problem is you are missing these versions in both compass-style.org and the mark down file

0.12.7 July 17, 2014 (1.11 MB)
0.12.6 April 19, 2014 (1.11 MB)
0.12.5 April 7, 2014 (1.11 MB)
0.12.4 March 20, 2014 (1.03 MB)
0.12.3 March 4, 2014 (1.11 MB)

That's a lot of changes to be missing

**Comment by scottdavis** [comment]
You are 100% correct they should have been in there but, sometimes as the nature of humans people forget or they want to get a release out to get bug fixes done and I apologize for any grief it may have caused you.

**Comment by joshuambg** [comment]
Thanks. Not trying to cause a fuss :-/ Just letting you know about the inconsistency. Look forward to seeing these versions added to history in the next update to Change Log and mark down file!
